src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -6,19 +6,20 @@
   position: absolute;
   background: $primary;
   top: 66px;
-  left: 50%;
+  // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
+  left: 40%;
   width: 500px;
```

Comment:
You say it takes 20%, how about this width value?

Reply:
Looks like it has 500px width and -250 margin.

Reply:
I believe this is a technique to make the bar centered. This width + margin styles set the `moving point` of the bar to the center of the bar